engine/servermaster/server.go

```diff
@@ -504,13 +506,29 @@ func (s *Server) registerMetaStore(ctx context.Context) error {
 	if err := s.metaStoreManager.Register(cfg.FrameMetaConf.StoreID, cfg.FrameMetaConf); err != nil {
 		return err
 	}
+	if cfg.FrameMetaConf.StoreType == metaModel.StoreTypeSQL {
+		// Normally, a schema will be created in advance and we may have no privilege
+		// to create schema for framework meta. Just for easy test here. Ignore any error.
+		ctx, cancel := context.WithTimeout(ctx, 3*time.Second)
+		defer cancel()
+		if err := meta.CreateSchemaIfNotExists(ctx, *(s.cfg.FrameMetaConf)); err != nil {
+			log.Warn("create schema for framework metastore fail, but it can be ignored.",
+				zap.String("schema", s.cfg.FrameMetaConf.Schema),
+				zap.Error(err))
+		}
```

Comment:
If this schema doesn't exist and engine has no privilege here, when and where will the error be raised.

Reply:
The error will be raised when creating `framework meta client` and the server-master will exits correspondingly.